### ISSUE TITLE
Question: Calculate date values

### ISSUE THREAD

**Issue by amo42** [rezo-labs/directus-extension-computed-interface#13]
Hey :) 

First: awesome extension, thanks.

I've got a question before testing in the field. Does the extension also work with dates? So I saw I can parse values to ISO Dates, but would it be possible to use those for something like `SUBTRACT("2022-11-10", "2022-11-08")` expect to be `2`?

Best regards
Andreas

**Comment by duydvu** [comment]
Hi @amo42 

This is not possible at the moment since I forgot to add the date conversion operator :). It should be easy so it will be added in the next release.

Thanks for reminding me :)

**Comment by amo42** [comment]
Hi @duydvu may I ask how to use it along with date fields? I've installed the new version but always get 0 as a result. Regardless of the usage of fields or hard coded iso date strings:

I've tried the following:
- {{ SUBTRACT(DATE(ende), DATE(start) ) }}
- {{ SUBTRACT(DATE(DATE_ISO(ende)), DATE(DATE_ISO(start)) ) }}
- {{ SUBTRACT(DATE("2022-11-16")), DATE("2022-11-10") ) }}


The collection does have the two fields "start" and "ende". the use case would be to have the days between two given dates (start and end)
